RCTAppleHealthKit/RCTAppleHealthKit+Utils.m

```diff
@@ -358,10 +358,14 @@ + (NSString*)stringForHKWorkoutActivityType:(int) enumValue{
             return @"Basketball";
         case HKWorkoutActivityTypeBowling:
             return @"Bowling";
+        HKWorkoutActivityTypeCardioDance:
```

Comment:
We should have a `case` condition on this one. So it would be as

```Objective-C
...
case HKWorkoutActivityTypeCardioDance:
  return @"CardioDance";
...
```

Reply:
Fixed and moved CardioDance to the correct place in the list, in alphabetical order.